inlong-sort/sort-connectors/pulsar/src/main/java/org/apache/inlong/sort/pulsar/table/DynamicPulsarDeserializationSchema.java

```diff
@@ -132,6 +140,40 @@ public void open(DeserializationSchema.InitializationContext context) throws Exc
 
     }
 
+    /**
+     * reflect get metricGroup
+     *
+     * @param context Contextual information that can be used during initialization.
+     * @return metric group that can be used to register new metrics with Flink and to create a nested hierarchy based
+     *         on the group names.
+     */
+    private MetricGroup getMetricGroup(DeserializationSchema.InitializationContext context)
+            throws NoSuchFieldException, IllegalAccessException {
+        MetricGroup metricGroup;
+        String className = "RuntimeContextDeserializationInitializationContextAdapter";
```

Comment:
pls extract constant for class name

Reply:
I don't think that it need constant for class name.